index.js

```diff
@@ -298,14 +750,32 @@ function createSession(user) {
  */
 function findWebClient(circuit) {
   return circuit.getDevices().then(devices => {
-      return devices.find(device => {
-          return (device.clientId !== circuit.user.clientId) &&
-              ((device.clientInfo.deviceType === 'WEB') ||
-              (device.clientInfo.deviceType === 'APPLICATION' && device.clientInfo.deviceSubtype === 'DESKTOP_APP'));
-      });
+    return devices.find(device => {
+      return device.clientId !== circuit.user.clientId && (device.clientInfo.deviceType === 'WEB' || (device.clientInfo.deviceType === 'APPLICATION' && device.clientInfo.deviceSubtype === 'DESKTOP_APP'));
+    });
   });
 }
 
+/**
+ * Search for a user
+ */
+async function getUsers(circuit, conv, user) {
```

Comment:
The "user" parameter is really a search string, not a user object. Would call it "query" or something like it.

Reply:
Done.